addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -81,7 +81,7 @@ def _fetch_query(self, query, fields):
         records = super()._fetch_query(query, fields)
         if self.env.context.get('hide_employee_name') and 'employee_id' in self.env.context.get('group_by', []):
             self.env.cache.update(records, self._fields['name'], [
-                record.name.split(':')[-1].strip()
+                (record.name.split(':')[-1].strip() if record.name else record.name)
```

Comment:
is it working with else part? that might fail when name  has boolean value :thinking: